src/plugins/saved_search/public/plugin.ts

```diff
@@ -147,3 +153,15 @@ export class SavedSearchPublicPlugin
     };
   }
 }
+
+registerSavedObjectToPanelMethod(SavedSearchType, (savedObject) => {
```

Comment:
Nit: it seems a bit odd to me to have the registration as a side effect of the plugin file being imported. While not necessary, I wonder if the intent would be clearer if `registerSavedObjectToPanelMethod` was exposed through the embeddable plugin contract, and consumers would register in their plugin `setup` or `start` methods?

Reply:
@ThomThomson expressed this preference—perhaps he can chime in to explain.

Reply:
Yes I did express this preference. In my opinion because plugins are stateful, most of what is currently accomplished by passing around plugin start contracts could be accomplished using functions exported directly from the bundle.

This is simpler because: 
- In your `kibana.jsonc` you can use a `requiredBundle` instead of a `requiredPlugin` for the dependency.
- You can import the function directly to where it's used rather than having to prop drill or pass the plugin start contract through to where it's required.

The new Embeddable registry is handled this way, and works quite well so I recommended that @drewdaemon follow that pattern for this registry as well. 

I'd like to point out that it isn't _necessary_ to have the call to `registerSavedObjectToPanelMethod` happen at the top level of the plugin file like this, it would be equally valid to call that function from inside the `setup` or `start` methods.